sdk/core/azure-core/src/main/java/com/azure/core/serializer/JsonSerializer.java

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.serializer;
```

Comment:
We already have a package named `com.azure.core.util.serializer`. Should we move this interface to that package instead of creating this new package?

Reply:
Doesn't hurt to move it into that package, the package currently contains Jackson deserializer/serializer implementations so I don't know if we want to mix this interface into there.

Reply:
Moved into `util.serializer`.